common/lib/capa/capa/capa_problem.py

```diff
@@ -555,7 +567,10 @@ def get_html(self):
         Main method called externally to get the HTML to be rendered for this capa Problem.
         """
         self.do_targeted_feedback(self.tree)
-        html = contextualize_text(etree.tostring(self._extract_html(self.tree)), self.context)
+        html_tree = self._extract_html(self.tree)
+        # don't include root div tags
```

Comment:
Without this change resulting HTML will be

```XML
<div class="problem">
    <div>
        <div class="question", id="0"></div>
        <div class="question", id="1"></div>
   </div>
</div>
```

With this change resulting HTML will be and this is what we want

```XML
<div class="problem">
    <div class="question", id="0"></div>
    <div class="question", id="1"></div>
</div>
```